exoscale/datasource_exoscale_domain_record_test.go

```diff
@@ -0,0 +1,156 @@
+package exoscale
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+var (
+	testAccDataSourceDomainRecordConfigCreate = fmt.Sprintf(`
+	resource "exoscale_domain_record" "mx2" {
+	  domain      = "${exoscale_domain.exo.id}"
+	  name        = "%s"
+	  record_type = "%s"
+	  content     = "%s"
+	  prio        = %d
+	  ttl         = %d
+	}
+	`,
+		testAccResourceDomainRecordNameUpdated,
```

Comment:
Please declare `testAccDatasourceDomainRecord*` variables dedicated to this data source's tests instead of reusing other tests' variables, i.e. as done here: https://github.com/terraform-providers/terraform-provider-exoscale/pull/34/files#diff-99d9853c3472dd66f6f7830eb6b6a3f7R12

Reply:
👍